### DISCUSSION TITLE
Netstream.seek(seconds:Number) not working in Android

### DISCUSSION THREAD

**Discussion by toysonlocation** [airsdk/Adobe-Runtime-Support#2155] [Q&A]
Before I consider it a bug, I'm wondering if anyone has been able to do arbitrary .seek(secs) on a Netstream in Android? I'm using a local mp4 that is 100% keyframes, and I can't seek to anywhere in the video but (0). It works as expected on iOS.

I'm using this in StageVideo. I need to have the video beneath other DisplayObjects, so any ANE suggestions would have to allow for the video to be played beneath Stage children.

**Comment by toysonlocation** [comment]
Any insights or intuitions are highly appreciated!

**Reply by ajwfrost** [reply]
I wasn't aware of any bugs around seek on Android.. but this may be one by the sounds of it! Are you able to create a really basic/standalone test case that demonstrates the problem and we can try it out here?

thanks
